homeassistant/components/sms/config_flow.py

```diff
@@ -2,17 +2,30 @@
 import logging
 
 import gammu  # pylint: disable=import-error
-import voluptuous as vol
+import voluptuous as vol  # pylint: disable=wrong-import-order, import-error
 
 from homeassistant import config_entries, core, exceptions
 from homeassistant.const import CONF_DEVICE
-
-from .const import DOMAIN
+from homeassistant.helpers import selector
+
+from .const import (  # pylint: disable=unused-import
```

Comment:
All of them are used? Why is this pylint check disabled?

Reply:
> All of them are used? Why is this pylint check disabled?

Resolved